### DISCUSSION TITLE
sometimes can't received task,but task already sent

### DISCUSSION THREAD

**Discussion by banzhihang** [celery/celery#7180] [Q&A]
i have tasks that are already sent but will not received. This the output of celery worker:

```
Scheduler: Sending due task attendance_1030 (login.tasks.auto_attendance_1030)
[2021-07-22 22:45:00,087: DEBUG/MainProcess] login.tasks.auto_attendance_1030 sent. id->778a3b48-ea33-49dd-9ced-742e309a41f3
[2021-07-22 22:45:00,087: DEBUG/MainProcess] beat: Waking up in 5.00 minutes.
```

```l
celery worker -A Clock --loglevel=info --pool=solo -n main
```

```
celery version is 4.4.2,broker is rabbitmq
```

[a comment by an automated account is not shown]

**Comment by syrull** [comment]
Could you follow the instructions of reporting here: https://github.com/celery/celery/blob/master/CONTRIBUTING.rst#id3

Without any code and the proper logs and reports I can't really do much for you. Also this looks like the celery-beat's log not the workers one.

**Comment by auvipy** [comment]
besides you should try a more recent version of celery or master to verify

**Comment by thedrow** [comment]
If the task is sent but not received, check which queue it is published on.
It is very likely your worker isn't consuming from it.
